python/ray/ray_perf.py

```diff
@@ -78,7 +80,9 @@ def timeit(name, fn, multiplier=1):
 
 def main():
     print("Tip: set TESTS_TO_RUN='pattern' to run a subset of benchmarks")
-    ray.init()
+    cluster = Cluster(initialize_head=True, connect=True, head_node_args={"object_store_memory":20*1024*1024*1024, "num_cpus":8})
+    client_node = cluster.add_node(object_store_memory=20*1024*1024*1024, num_gpus=1, num_cpus=8)
```

Comment:
This will have a hard to guess impact on the other microbenchmarks. Can you make this a separate file?